src/components/common/ImageRequire.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <img
+    class="responsive-image"
+    :src="require(`../../assets/${urlImage}`)"
+    :alt="altImage"
+  />
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from "vue-property-decorator";
+
+@Component({
+  name: "ImageRequire",
+})
+export default class ImageRequire extends Vue {
+  @Prop(String) private urlImage: string;
+  @Prop(String) private altImage: string;
+}
+</script>
+
+<style scoped lang="scss">
+.responsive-image {
```

Comment:
@JuanJosePG I'm not sure if this name is correctly... maybe for a common class is fine, but for a component....